src/plugins/intel_gpu/src/kernel_selector/kernels/group_normalization/group_normalization_kernel_ref.cpp

```diff
@@ -0,0 +1,170 @@
+// Copyright (C) 2023 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "group_normalization_kernel_ref.h"
+#include <kernel_selector_utils.h>
+
+namespace kernel_selector {
+
+ParamsKey GroupNormalizationKernelRef::GetSupportedKey() const {
+    ParamsKey k;
+    k.EnableInputDataType(Datatype::F16);
+    k.EnableInputDataType(Datatype::F32);
+    k.EnableOutputDataType(Datatype::F16);
```

Comment:
Int8 support is available?

Reply:
It's not since the [documentation](https://docs.openvino.ai/nightly/openvino_docs_ops_normalization_GroupNormalization_12.html) says that all inputs of the operation support only **any supported floating point type**.